libs/perun/dialogs/src/lib/show-notification-history-dialog/show-notification-history-dialog.component.html

```diff
@@ -14,10 +14,10 @@ <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATION_LIST.TITLE' | customTranslate | tra
   >
 </div>
 <div mat-dialog-actions>
-  <button (click)="onClear()" class="ms-auto" mat-stroked-button>
-    {{'DIALOGS.NOTIFICATION_LIST.CLEAR' | customTranslate | translate}}
-  </button>
-  <button (click)="onCancel()" class="ms-2" mat-stroked-button>
+  <button (click)="onCancel()" class="ms-auto" mat-stroked-button>
     {{'DIALOGS.NOTIFICATION_LIST.CANCEL' | customTranslate | translate}}
   </button>
+  <button (click)="onClear()" class="ms-2" mat-stroked-button>
+    {{'DIALOGS.NOTIFICATION_LIST.CLEAR' | customTranslate | translate}}
+  </button>
```

Comment:
With this, I thought the clear notification button should have some colour.

Reply:
Sure, we can add `accent` color here,